core/EE_Config.core.php

```diff
@@ -2407,6 +2409,48 @@ public function set_default_max_ticket_on_EEM_Event()
     }
 
 
+    /**
+     * Sets the default consent checkbox text. This needs to be done a bit later than when EE_Registration_Config is
+     * constructed because that happens before we can get the privacy policy page's permalink.
+     *
+     * @throws InvalidArgumentException
+     * @throws \EventEspresso\core\exceptions\InvalidDataTypeException
+     * @throws \EventEspresso\core\exceptions\InvalidInterfaceException
+     */
+    public function setDefaultCheckboxLabelText()
+    {
+        if ($this->getConsentCheckboxLabelText() === null
+            || $this->getConsentCheckboxLabelText() === '') {
```

Comment:
was this not set during construction to "I Consent to Privacy Policy" ???

Reply:
oh right... this was getting overwritten with nothing after loading the config from the db eh?

Reply:
I think you're onto something, something was fishy. I think multiple iterations of this code got mixed up. 
We shouldn't set the default value during construction because this should be setting the default.